swagger/dfc.yaml

```diff
@@ -318,6 +318,7 @@ paths:
                     - "@id": http://test.host/api/dfc/enterprises/10000
                       "@type": dfc-b:Enterprise
                       dfc-b:hasAddress: http://test.host/api/dfc/addresses/40000
+                      dfc-b:hasPhoneNumber: 0404 444 000 200
                       dfc-b:websitePage: openfoodnetwork.org
```

Comment:
I see that the DFC builder accepts an array, but a single value is output here. I'm curious how do multiple values appear?

Of course, we don't need to worry about it because we don't have multiple values.

Reply:
Multiple values are put in an array. In the semantic web, that's the same as repeating the attribute several times:

- a has b
- a has c
- a has d

Same as "a has [b,c,d]". From the JSON you can't see if an attribute supports multiples or not. That's only defined in the ontology, I think.